vitrina/settings.py

```diff
@@ -484,3 +484,5 @@
 OTP_EMAIL_TOKEN_VALIDITY = 600
 
 USE_OTP_VALIDATION = env.bool("USE_OTP_VALIDATION", default=True)
+
+TRANSLATION_REQUEST_TIMEOUT = env("TRANSLATION_REQUEST_TIMEOUT", default=2)
```

Comment:
```suggestion
TRANSLATION_REQUEST_TIMEOUT = env("TRANSLATION_REQUEST_TIMEOUT", default=1)
```

Probably it will never start responding and even if starts - one second should be enough to respond.